frontend/src/scenes/authentication/Login.tsx

```diff
@@ -49,10 +52,21 @@ export function Login(): JSX.Element {
     const { precheck } = useActions(loginLogic)
     const { precheckResponse, precheckResponseLoading, login, isLoginSubmitting, generalError } = useValues(loginLogic)
     const { preflight } = useValues(preflightLogic)
+    const { featureFlags } = useValues(featureFlagLogic)
 
     const passwordInputRef = useRef<HTMLInputElement>(null)
     const isPasswordHidden = precheckResponse.status === 'pending' || precheckResponse.sso_enforcement
 
+    useEffect(() => {
+        try {
+            if (featureFlags[FEATURE_FLAGS.AUTO_REDIRECT]) {
+                redirectIfLoggedInOtherInstance()
+            }
+        } catch (e) {
+            console.error('unable to redirect to logged in instance', e)
```

Comment:
We should probably not just `console.error()` but send to Sentry (`captureException()`)